### ISSUE TITLE
[BUG]: Bug in Eureka Seven - Vol.2 The new vision - Mission Search 505

### ISSUE THREAD

**Issue by Guilherme520** [PCSX2/pcsx2#10946]
### Describe the Bug

During the game after the Search 505 mission, there is a task in story mode that requires you to talk to an NPC in the game who is at the top of the ladder, but it is not possible to climb the ladder to be able to talk to the WES NPC.
<img width="543" alt="Captura de tela 2024-03-19 000322" src="https://github.com/PCSX2/pcsx2/assets/163958312/05bbbbd7-35f9-4301-b325-cf52c82fd88e">

### Reproduction Steps

There is no abnormal event, the character just does not climb the ladder leading to the dialogue with the NPC

### Expected Behavior

The game is perfect, just needed to see how to continue the game's progress after this mission and and see ways to skip cut scenes that are a little long.
<img width="959" alt="Captura de tela 2024-03-19 002721" src="https://github.com/PCSX2/pcsx2/assets/163958312/0ab3a497-fb71-4b14-935b-15520928364d">

### PCSX2 Revision

V1.7.5623

### Operating System

Windows 10 (64bit)

### CPU

Ryzen 5 5600G

### GPU

Radeon Graphics Vega 7

### GS Settings

no hacks or any other changes

<img width="607" alt="Captura de tela 2024-03-19 003516" src="https://github.com/PCSX2/pcsx2/assets/163958312/6ab6393b-d3c8-45cb-a6da-083fba5ae04d">
<img width="540" alt="Captura de tela 2024-03-19 005531" src="https://github.com/PCSX2/pcsx2/assets/163958312/7d49626e-e919-4415-a38c-18c2f4bfbd25">

### Emulation Settings

<img width="601" alt="Captura de tela 2024-03-19 011038" src="https://github.com/PCSX2/pcsx2/assets/163958312/88049c15-bd61-4a8d-815d-a1ca35962997">

**Comment by JordanTheToaster** [comment]
Can you post a savestate in the spot please.

**Comment by Guilherme520** [comment]
like this?

<img width="569" alt="Captura de tela 2024-03-19 012908" src="https://github.com/PCSX2/pcsx2/assets/163958312/29a828b8-fd83-4730-8d7b-bb0f080f526e">

**Comment by JordanTheToaster** [comment]
> like this?
> 
> <img alt="Captura de tela 2024-03-19 012908" width="569" src="https://private-user-images.githubusercontent.com/163958312/313920516-29a828b8-fd83-4730-8d7b-bb0f080f526e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA4MjI5MzUsIm5iZiI6MTcxMDgyMjYzNSwicGF0aCI6Ii8xNjM5NTgzMTIvMzEzOTIwNTE2LTI5YTgyOGI4LWZkODMtNDczMC04ZDdiLWJiMGYwODBmNTI2ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxOVQwNDMwMzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02OTEwZDk1N2Q1ZTBhMmQzYTJlNjZlNGQwOTU1OWRkNGY0NzFkNWEwYzM2ZmEwODE5ODFmMzU0MDEzYTA0NTQzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.YG2xtlxoH5O7JyvLeC4Sx0D1cgkJ7QiJt2a2wboHEqg">

The file itself upload it here.

**Comment by Guilherme520** [comment]
how do you do it?

**Comment by JordanTheToaster** [comment]
> how do you do it?

Documents folder PCSX2 sstates folder post the savestate file for the game from there here.

**Comment by Guilherme520** [comment]
like this?

**Comment by Guilherme520** [comment]
don’t support that file type

**Comment by Guilherme520** [comment]
don’t support that file type

**Comment by TheTechnician27** [comment]
Yeah, Github doesn't support the format that PCSX2 saves the save state in. You'll need to zip it first. If you're on Windows, this can be done by right-clicking the file and then select 'Send to' and then from there select 'Compressed folder'. That will create a .zip file, which can be uploaded to Github. :)

**Comment by Guilherme520** [comment]
Now go :)

[SLUS-21538 (EAF91930).02.zip](https://github.com/PCSX2/pcsx2/files/14645336/SLUS-21538.EAF91930.02.zip)

**Comment by TheTechnician27** [comment]
Thanks a bunch, Guil! Hopefully Jordan is able to get this figured out.

**Comment by Guilherme520** [comment]
I would like to thank you for your attention, I really hope you can resolve it without having too many problems.

**Comment by TheTechnician27** [comment]
> I would like to thank you for your attention, I really hope you can resolve it without having too many problems.

We massively appreciate you contributing too; so much gets fixed every day because users like you report these issues.

**Comment by JordanTheToaster** [comment]
> Now go :)
> 
> [SLUS-21538 (EAF91930).02.zip](https://github.com/PCSX2/pcsx2/files/14645336/SLUS-21538.EAF91930.02.zip)

Go to Settings > Emulation and set your EE cycle rate to 50% and try to walk up the left hand side of the stairs that should work for some reason.

**Comment by Guilherme520** [comment]
It seems like a lie but it worked, thank you very much!

![image](https://github.com/PCSX2/pcsx2/assets/163958312/10b1cb23-19cd-4f0d-8238-068df551b1da)

**Comment by JordanTheToaster** [comment]
> It seems like a lie but it worked, thank you very much!
> 
> ![image](https://private-user-images.githubusercontent.com/163958312/313929157-10b1cb23-19cd-4f0d-8238-068df551b1da.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA4MjYxODQsIm5iZiI6MTcxMDgyNTg4NCwicGF0aCI6Ii8xNjM5NTgzMTIvMzEzOTI5MTU3LTEwYjFjYjIzLTE5Y2QtNGYwZC04MjM4LTA2OGRmNTUxYjFkYS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxOVQwNTI0NDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jYjM1YmQwMzcwNzhlMTkzNTk3MzNhODI3ZGE2ZTJmZGYwYzMwMzg2MTMzZjFkMGEwNmIwNmY5Zjk2NDRiMWM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.26YjoWybQ5XRASmHWUBqcvDD3PKgl6xqfH5QEVmTCdA)

Still need to figure out why this is happening so I'll leave the issue open for now.
